### DISCUSSION TITLE
An anti-war message

### DISCUSSION THREAD

**Discussion by uazo** [bromite/bromite#1842] [Ideas]
I hope that like me you too repudiate this absurd war and how the Russian public opinion does not express itself negatively about what they are doing.
for this I propose our "sanction": I would inhibit the use of bromite to devices in Russian language with a total block of use, my effort can only be used by people who love peace and freedom

**Comment by csagan5** [comment]
Bromite is just software; it can be used by good people and bad people, the browser does not make a test for that so it should not make a test for whatever belief or opinion the developers have before allowing usage; regardless, there are also supporters of the war that do not speak Russian.

I have already planned yesterday to add a default favicon to support the people affected by war, that will suffice. For anyone else that would like to help: go find a reputable donation website, it is off-topic to discuss the on-going war here.

**Comment by ghost** [comment]
> I would inhibit the use of bromite to devices in Russian

Many Russians do not support The War.

**Comment by uazo** [comment]
> regardless, there are also supporters of the war that do not speak Russian.
> it is off-topic to discuss the on-going war here
> Many Russians do not support The War.

yes, sure, I exaggerated

> I have already planned yesterday to add a default favicon to support the people affected by war

that's fine, I fully support you

**Comment by albotroz** [comment]
Do you know what an a antiwar message means?
"No war - Peace!"

This is an a antiwar message. And not your book burning 2.0 provocation.
And Bromite's oneside political statement "Support Ukraine" is also not an a antiwar-statement. It's a war statement. To prolonging this destructive war.

Crazy people everywhere...
Seriously don't know what it is wrong with this generation. It seems like you don't know what you talking about when you write such "genius " ideas of so called antiwar-statements.

If you now hate every Russian and everything Russian then say it and don't use Russian software and Russian sourcecodes.

If you want to help Ukraine and make an "anti-war" statement, donate money to the european PEACE fund. This will bring more weapons for Ukraine. Because, as we all know, more weapons and more deaths lead to peace - isn't it? That's what you and yourselves are about when you talk about an alleged anti-war statement and "support for Ukraine".
Double standards and hypocrisy in the most disgusting form.

Nowhere is safe these days from you warmongers in the guise of supposed pacifists.
You politicize every corner and every area. 

You don't even know what antiwar-message means.
Look back 40 years and look at the anti-war protests. That was the definition of antiwar-messages.

Provocations and exclusions make the current situation even more dangerous. When the mushroom cloud appears on the horizon, your bromite is of no use to you either, regardless of Russian language support.
So grow up!

**Reply by csagan5** [reply]
> Do you know what an a antiwar message means? "No war - Peace!"
> 
> This is an a antiwar message. And not your book burning 2.0 provocation. And Bromite's oneside political statement "Support Ukraine" is also not an a antiwar-statement. It's a war statement. To prolonging this destructive war.

Have you taken the time to open [the link](https://vostok-sos.org/en/ukraine-under-fire-support-vostok-sos-aid-operation/) and check what type of support is mentioned there? Go give a look before posting:
> Vostok SOS is responding to the escalating situation with a comprehensive humanitarian campaign. We help people evacuate, and provide humanitarian aid and psychosocial support. We have hotlines open for affected people and our team is on the ground in the region, ready to coordinate aid.



> If you now hate every Russian and everything Russian then say it and don't use Russian software and Russian sourcecodes.

No such thing here. I explained very well why language and ethnicity is not important, go read [my comment](https://github.com/bromite/bromite/discussions/1842#discussioncomment-2256837). If you prefer to post without reading all the comments then we can spare your contributions here.

> Look back 40 years and look at the anti-war protests. That was the definition of antiwar-messages.

You are referring to American citizens opposing to their own country going to war in Vietnam. Those were (political) anti-war protests aimed at blocking the draft. The message added to Bromite has the only goal to support the Ukrainian population in distress, not to block the draft in Russia. Russians can make their own messaging and protests for that.

**Comment by csagan5** [comment]
Also: I did not add anything to Bromite **in response** to this discussion. I was already adding a **support** message long before this discussion was opened. The original poster wanted to add an anti-war message and wanted to make it specific to Russian speakers, not me.

Anyone can discuss anything (on topic) here but please be mindful of the [house rules](https://github.com/bromite/bromite/discussions/893).
